todo-mvc-kitchensink/.dojorc

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
This doesn't look right, which cli command is this supposed to be the config form?